src/main/java/Event.java

```diff
@@ -0,0 +1,26 @@
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class Event extends Task {
+    private Date timeL;
+    private Date timeR;
+    private static final SimpleDateFormat inputFormat = new SimpleDateFormat("dd-MM-yyyy");
+    private static final SimpleDateFormat outputFormat = new SimpleDateFormat("dd MMM yyyy");
+
+    public Event(String description, String timeL, String timeR) throws ParseException {
+        super(description);
+        this.timeL = inputFormat.parse(timeL);
+        this.timeR = inputFormat.parse(timeR);
+    }
+
+    @Override
+    public String toString() {
+        return "[E]" + super.toString() + " (from: " + outputFormat.format(timeL) + " to: " + outputFormat.format(timeR) + ")";
```

Comment:
Hi, can possibly consider use of a new line to break this return statement into two if it is more than character limit?